Network/Network.cpp

```diff
@@ -486,15 +486,15 @@ namespace WPEFramework
                     string interface = "";
                     bool persist = false;
 
-                    getStringParameter("interface", interface)
+                    getDefaultStringParameter("interface", interface, "")
 
                         if (!(strcmp (interface.c_str(), "ETHERNET") == 0 || strcmp (interface.c_str(), "WIFI") == 0))
                         {
                             LOGERR ("Call for %s failed due to invalid interface [%s]", IARM_BUS_NETSRVMGR_API_setDefaultInterface, interface.c_str());
                             returnResponse (result)
                         }
 
-                    getBoolParameter("persist", persist)
+                    getDefaultBoolParameter("persist", persist, false)
```

Comment:
This is already being done outside. I understand it's good to do it within this getDefaultBoolParameter API but checking how does this help with the crash.